tests/backbeat/tests/api/metrics.js

```diff
@@ -99,66 +144,48 @@ describe.skip('Backbeat replication metrics', function dF() {
                         assert(byteResult - prevDataBytes === 100);
 
                         const throughputSize = body.throughput.results.size;
-                        assert(throughputSize > 0);
+                        assert(throughputSize >= 0);
```

Comment:
Maybe you can use a unique key name for this test (assuming it maintains the common key prefix) such that you can guarantee the expected throughput size.

Reply:
As per our conversation, the object-level metrics don't apply here, so the above comment is `void` 😂.